install/upgrade.php

```diff
@@ -204,6 +204,65 @@ protected function upgrade_2_7() {
 	 * @return void
 	 */
 	protected function upgrade_3_4() {
+		global $wpdb;
+
 		delete_transient( 'pll_languages_list' );
+
+		if ( ! defined( 'POLYLANG_PRO' ) || ! POLYLANG_PRO ) {
+			return;
+		}
```

Comment:
We don't even need this as we just won't get any meta. Also removing it would account for the (improbable) case that some one does Polylang Pro -> Polylang -> Polylang Pro and updates while using Polylang.

Reply:
Removed in [7eb63dd](https://github.com/polylang/polylang/pull/1192/commits/7eb63dddb12251bec3b3174a05286563bf774f77).